insights/parsers/sysconfig.py

```diff
@@ -458,6 +461,36 @@ class PrelinkSysconfig(SysconfigOptions):
     pass
 
 
+@parser(Specs.sysconfig_sshd)
+class SshdSysconfig(SysconfigOptions):
+    """
+    A parser for analyzing the ``/etc/sysconfig/sshd`` configuration file.
+
+    Sample Input::
+        # Configuration file for the sshd service.
+
+        # The server keys are automatically generated if they are missing.
+        # To change the automatic creation, adjust sshd.service options for
+        # example using  systemctl enable sshd-keygen@dsa.service  to allow creation
+        # of DSA key or  systemctl mask sshd-keygen@rsa.service  to disable RSA key
+        # creation.
+
+        # System-wide crypto policy:
+        # To opt-out, uncomment the following line
+        # CRYPTO_POLICY=
+        CRYPTO_POLICY=
```

Comment:
@jsvob would you mind to look at this in the rendered docs, it doesn't look correct.

Reply:
Thank you for spotting the bug!